public/jsx/components/card-form.jsx

```diff
@@ -20,7 +21,7 @@ module.exports = React.createClass({
         {
           'data-braintree-name': 'cardholder_name',
           'id': 'cardholder',
-          'placeholder': 'Card holder name',
+          'placeholder': gettext('Card holder name'),
```

Comment:
I don't see this in @brampitoyo mocks https://github.com/mozilla/payments/pull/55/files